packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -85,6 +88,17 @@ export default class VaDropdown extends mixins(
     }
   }
 
+  get stateClasses (): string[] {
+    const baseClass = 'va-dropdown'
+    const classes: string[] = []
+
+    if (this.$props.disabled) {
+      classes.push(`${baseClass}_disabled`)
```

Comment:
Looks very hacky. What the intention is here?
Generally computeds should not have side effects.

Reply:
Completely agree that Computed and Getters should not have any side effects.

But I believe there was a misunderstanding because I do not see any side effects here.
```
  get classComputed (): string[] {
    const baseClass = 'va-dropdown'
    const classes: string[] = []

    if (this.$props.disabled) {
      classes.push(`${baseClass}--disabled`)
    }

    return classes
  }
```

It's just a local constant which is used for convinience.

Most of our components are using the same approach - binding classes list Object or Array.

I could change it to something like this:

```
get classComputed () {
    return { 
      va-dropdown--disabled: this.$props.disabled
    }
}
```

From my perspective though current version is better for a possible component expansion in the future.